buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -23,28 +20,7 @@ object Config {
     // https://github.com/JLLeitschuh/ktlint-gradle/releases
     const val KTLINT_GRADLE_VERSION = "10.2.1"
 
-    // https://github.com/pinterest/ktlint/blob/master/CHANGELOG.md
```

Comment:
@BottleRocket-Colin I definitely understand why you've cleaned house on a majority of entries listed here (they aren't being used) and appreciate the effort!

That being said, I do think there is value to having a few of the buildscript/apply plugins (`detekt` and `gradle versions`) added back in some future PR. I'll add it to our todo list